module/VuFind/src/VuFind/Controller/OAuth2Controller.php

```diff
@@ -250,15 +250,15 @@ public function authorizeAction()
             // Store OpenID nonce (or null if not present to clear any existing one)
             // in the access token table so that it can be retrieved for token or
             // user info action:
-            $this->accessTokenTable
+            $this->accessTokenService
```

Comment:
We can now safely rely on `$user` implementing `UserEntityInterface`, so maybe we should start using getters here (e.g. replace `$user->id` with `$user->getId()`.